UnityProject/Assets/Scripts/UI/Objects/Medical/genetics/MutationMiniGameElement.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using UI.Core.NetUI;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class MutationMiniGameElement : DynamicEntry
+{
+	public  BodyPartMutations.MutationRoundData.SliderParameters SliderParameters;
+
+	public MutationUnlockMiniGame MutationUnlockMiniGame;
+
+	public NetServerSyncString netServerSyncString;
+
+	public NetSlider SliderLever;
+
+	public NetSlider Indicator;
+
+	public Image LineImage;
+
+	public bool locked = false;
+
+	//net Service Synchronise string
+	//Some hooks into the sliders to update values on client and server
+	public void SetValues(BodyPartMutations.MutationRoundData.SliderParameters InSliderParameters , MutationUnlockMiniGame InMutationUnlockMiniGame)
+	{
+		SliderParameters = InSliderParameters;
+		MutationUnlockMiniGame = InMutationUnlockMiniGame;
+		netServerSyncString.SetValue(JsonConvert.SerializeObject(InSliderParameters));
+		AccumulatedForces.Clear();
+		locked = false;
+	}
+
+	public void Awake()
+	{
+		netServerSyncString.OnChange.AddListener(Setup);
+		Indicator.GetComponent<Slider>().onValueChanged.AddListener(ValueChangeSlider);
+
+		//PreviouslyValueLever = Indicator
```

Comment:
here as well, don't comment out code unless you write an explanation as to why its there and is at the end of the comment so you dont trigger codacy